src/cli/java/org/commcare/util/screen/QueryScreen.java

```diff
@@ -110,7 +111,7 @@ public String buildUrl(boolean skipDefaultPromptValues) {
             QueryPrompt prompt = userInputDisplays.get(key);
             for (String value : queryParams.get(key)) {
                 if (prompt != null && prompt.isSelect()) {
```

Comment:
@shubham1g5 I notice that in `FormplayerQueryScreen` you removed the `prompt.isSelect()` check (https://github.com/dimagi/formplayer/commit/4bc37f2bc6f87d6ad37f02c28ebb68ebbb0a0114#diff-d43851ef662a8e31af133ace2b4c60e87e8df0f14f8b70293ec884762f183518L33) should that have been done here too?

Reply:
yes, we should remove it from here as well.